### PR TITLE
Update NFT flow

### LINKED ISSUE #3093: Introduce NFT proceedings go to channel balance as part of the flow
- [ ] Update copy in the “Heads up!” Banner component, displayed under NFT options section, in the Video workspace (https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=3223%3A161358),
- [ ] Update description of the Form field component for NFT royalties (https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=3223%3A161045),
- [ ] Add a Banner component to the 3rd step of the NFT workspace (https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=2393%3A82183),
- [ ] Add a confirmation dialog for minting NFT for a video (https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=3692%3A140829).

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1202736679778647) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #3093

### CODE REVIEW

File: packages/atlas/src/api/queries/fragments.graphql [hunk 1]
```diff
@@ -192,9 +192,13 @@ fragment BasicNftFields on OwnedNft {
   creatorRoyalty
   lastSaleDate
   lastSalePrice
+  isOwnedByChannel
   ownerMember {
     ...BasicMembershipFields
   }
+  transactionalStatusUpdates {
+    __typename
+  }
```
Comment (kdembler) [inline packages/atlas/src/api/queries/fragments.graphql]: I think this isn't needed anymore, right?

File: packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx [hunk 2]
```diff
@@ -5,8 +5,10 @@ import { FC, PropsWithChildren } from 'react'
 import { BasicMembershipFieldsFragment } from '@/api/queries'
 import { NumberFormat } from '@/components/NumberFormat'
 import { Text } from '@/components/Text'
+import { SvgAlertsInformative24 } from '@/components/_icons'
 import { useMemberAvatar } from '@/providers/assets'
 import { formatDateTime } from '@/utils/time'
+import { RevenueBanner } from '@/views/global/NftSaleBottomDrawer/NftForm/NftForm.styles'
```
Comment (kdembler) [inline packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/AcceptTerms/AcceptTerms.tsx]: This should be relative import

File: packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx [hunk 3]
```diff
@@ -482,7 +499,7 @@ export const VideoForm: FC<VideoFormProps> = memo(({ onSubmit, setFormStatus })
         },
         disabled: videoFieldsLocked,
       }}
-      description="Royalties lets you earn commission from every sale of this NFT."
+      description="Disabling the comments section does not allow for posting new comments under this video and hides any existing comments made in the past."
```
Comment (kdembler) [inline packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx]: Why is this description used for royalties?

**Review by kdembler — CHANGES_REQUESTED** [review]

File: packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx (hunk 3 above)
Comment (rafalpawlow) [reply packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx]: lol, for some reason it was in the figma 😛

Zrzut ekranu 2022-09-5 o 15 39 38

**Review by kdembler — APPROVED** [review]

**Pull request merged**